base/db/R/convert.input.R

```diff
@@ -737,7 +737,7 @@ convert.input <-
     log_format_df = function(df){
       df  %>%
         format() %>%
-        rbind(colnames(.), .) %>%
+        rbind(colnames(.data$.), .data$.) %>%
```

Comment:
I think the `.` can be replaced with just `.data`.  Are there no tests to run this on to check? 

``` r
df %>% format() %>% rbind(colnames(.data), .data) %>%...
```

Reply:
All the tests are passing before changing to `df %>% format() %>% rbind(colnames(.data), .data) %>%...`

Reply:
Whelp, that's not great because pretty sure what you had there was not valid R code.  Not your fault that there aren't tests for that function though.

You could also re-write this to not use the pipe.  Remember that `a %>% f(.)` is the same as `f(a)`